src/development/platform-integration/platform-channels.md

```diff
@@ -84,21 +89,108 @@ messages happens automatically when you send and receive values.
 The following table shows how Dart values are received on the
 platform side and vice versa:
 
-| Dart                       | Java                | Kotlin      | Obj-C                                          | Swift                                   |
-| -------------------------- | ------------------- | ----------- | ---------------------------------------------- | --------------------------------------- |
-| null                       | null                | null        | nil (NSNull when nested)                       | nil                                     |
-| bool                       | java.lang.Boolean   | Boolean     | NSNumber numberWithBool:                       | NSNumber(value: Bool)                   |
-| int                        | java.lang.Integer   | Int         | NSNumber numberWithInt:                        | NSNumber(value: Int32)                  |
-| int, if 32 bits not enough | java.lang.Long      | Long        | NSNumber numberWithLong:                       | NSNumber(value: Int)                    |
-| double                     | java.lang.Double    | Double      | NSNumber numberWithDouble:                     | NSNumber(value: Double)                 |
-| String                     | java.lang.String    | String      | NSString                                       | String                                  |
-| Uint8List                  | byte[]              | ByteArray   | FlutterStandardTypedData typedDataWithBytes:   | FlutterStandardTypedData(bytes: Data)   |
-| Int32List                  | int[]               | IntArray    | FlutterStandardTypedData typedDataWithInt32:   | FlutterStandardTypedData(int32: Data)   |
-| Int64List                  | long[]              | LongArray   | FlutterStandardTypedData typedDataWithInt64:   | FlutterStandardTypedData(int64: Data)   |
-| Float32List                | float[]             | FloatArray  | FlutterStandardTypedData typedDataWithFloat32: | FlutterStandardTypedData(float32: Data) |
-| Float64List                | double[]            | DoubleArray | FlutterStandardTypedData typedDataWithFloat64: | FlutterStandardTypedData(float64: Data) |
-| List                       | java.util.ArrayList | List        | NSArray                                        | Array                                   |
-| Map                        | java.util.HashMap   | HashMap     | NSDictionary                                   | Dictionary                              |
+{% samplecode type-mappings %}
```

Comment:
Is there a way to preview this to make sure it actually worked as I intended?

Reply:
Yes, it's described in the README. We use docker for staging and it's easiest if the PR author stages it locally. However, this looks pretty good. I will accept and we can fix if there's an issue.

Reply:
The C++ tab doesn't work :( The URL it creates is wrong, presumably because of + being a special character. Is this a known problem with a workaround?

Reply:
Speculative fix at https://github.com/flutter/website/pull/6524 that needs testing in a staging environment.